docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I am a bit confused about the expected user interaction for doc generation with both the `Makefile` here and the `tox` step - can a developer just do `tox -e docs`?